components/tabBarComponents/EventTabBar.tsx

```diff
@@ -51,6 +51,16 @@ const styles = StyleSheet.create({
   icon: {
     alignItems: "center",
     justifyContent: "center"
+  },
+  plusIcon: {
+    alignItems: "center",
+    justifyContent: "center",
+    width: 50,
+    height: 50,
+    top: "4%",
+    left: "4%",
+    borderRadius: 15,
+    backgroundColor: "black"
```

Comment:
This seems hacky, maybe refer to the designs since those values are definitely not reflective of the them.